lisa/conf.py

```diff
@@ -56,7 +57,14 @@ class KeyDescBase(abc.ABC):
     to sanitize user input and generate help snippets used in various places.
     """
     INDENTATION = 4 * ' '
+    _VALID_NAME_PATTERN = r'^[a-zA-Z0-9-]+$'
```

Comment:
No underscores? I guess that's fine since we go with dash instead.

Reply:
yes, dashes are in use and I'd rather avoid having a mix of both, and the tentation for somebody to play with `setattr()` (if that somebody is motivated enough, they will do like in argparse).